sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OpenAiController.java

```diff
@@ -46,14 +53,10 @@ ResponseEntity<ResponseBodyEmitter> streamChatCompletionDeltas() {
     final var emitter = new ResponseBodyEmitter();
     final Runnable consumeStream =
         () -> {
-          final var totalOutput = new OpenAiChatCompletionOutput();
           // try-with-resources ensures the stream is closed
           try (stream) {
-            stream
-                .peek(totalOutput::addDelta)
-                .forEach(delta -> send(emitter, delta.getDeltaContent()));
+            stream.forEach(delta -> send(emitter, delta.getDeltaContent()));
```

Comment:
(Question)

Are you reducing the output to no longer print the usage stats? Just wondering, what's the rational?

Reply:
Compare here
https://github.com/SAP/ai-sdk-java/blob/4ebc2a34a00112e974961a13d525512f5cd3f09c/sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OpenAiController.java#L65-L77